docs/testplan/PFC_Congestion_Oversubscription_Test_Plan.md

```diff
@@ -0,0 +1,318 @@
+In either case, the ingress port VOQs, with not enough allocation granted for the lossless traffic streams causing the congestion, will in turn generate PFC pause (XOFF) messages out of the ingress port to temporarily stop the incoming lossless frames for the appropriate priority.
+
+#### Testbed Setup
+The test bed setup consists of three IXIA ports and a SONiC router as the device under test (DUT). All IXIA ports should have the same line speed.  The three DUT ports will be chosen at random across all the available ports in the router in order to eventually cover combinations of ports within the same line card and across line cards (or ports within the same ASIC vs. different ASICs on a single-card, multi-ASIC design).
+
+
+<p float="left">
+  <img src="Img/PFC_Testcase1.png" width="500"  hspace="250"/>
+</p>
+
+
+#### Test steps
+In this experiment we will create a total of four streams, with each of the two test ports sending two streams each. The aggregate bandwidth will be slightly over 100%:
+  *	The background traffic will include two lossy traffic streams, each with randomly chosen priorities (0..2, 5..7), and each having a 25% bandwidth.
+  *	The test data traffic will consist of two lossless traffic streams, with the SONiC default lossless priorities of 3 and 4, and randomized so that either both streams are at 30% bandwidth each, or for one stream to be at 25% while the other is at 30% bandwidth.
+
+The mapping of background and test data traffic streams to IXIA transmit ports can be randomized while keeping two streams per port; for example, one test port could send two lossless traffic streams and the second port two lossy streams, or each port could send two mixed streams, one lossless, one lossy.
+
+This experiment needs the following eight steps:
+1.	Start the background traffic from IXIA Tx ports 1 & 2; the combined line rate will be 50% of the egress port capacity (25% line rate bandwidth per stream)
+2.	Start the test traffic from IXIA Tx ports 1 & 2; the additional increase in bandwidth will be either 55% or 60% depending on randomization (one stream with 25% BW, second stream with 30%, or both at 30% each), for a total oversubscription of either 105% or 110%.
+3.	After a few seconds, measure the received bandwidth per flow on the IXIA Rx port 3: each of the four flows should have an allocation of 25% of the total egress port bandwidth. There should be PFC pause messages being received at the IXIA transmit ports causing the over-subscription.
+4.	Stop all traffic from the two IXIA ports 1 & 2
+5.	On the IXIA Rx port ensure that:
+a.	The received lossless traffic streams on IXIA port 3 show no missing and no out-of-order frames.
+b.	The lossy traffic streams got their fair share of allocated bandwidth and did not drop frames.

```

Comment:
"got their fair share" rate is not very clear here. Maybe you can say lossy traffic streams' RX rate equals to their TX rate.

Reply:
Updated.